src/omnisharp/launcher.ts

```diff
@@ -161,17 +161,26 @@ function launch(cwd: string, args: string[]): Promise<LaunchResult> {
     return PlatformInformation.GetCurrent().then(platformInfo => {
         const options = Options.Read();
 
+        let envVars = {};
+        if (options.useEditorFormattingSettings) 
+        {
+            let editorConfig = vscode.workspace.getConfiguration('editor');
+            envVars['formattingOptions:useTabs'] = !editorConfig.get('insertSpaces', true);
+            envVars['formattingOptions:tabSize'] = editorConfig.get('tabSize', 4);
+            envVars['formattingOptions:indentationSize'] = editorConfig.get('tabSize', 4);
+        }
```

Comment:
yes they can - in fact I believe `restore packages` is passed like that already if I recall correctly. I can change that for consistency, there is no real reason to use environment variables indeed